client/src/commons/Checkbox/CustomCheckbox.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { FormControlLabel, Checkbox, CheckboxProps } from '@material-ui/core';
+
+const CustomCheckbox: React.FC<Props> = (props: Props): JSX.Element => {
+
+    const { checkboxElements, checkboxesClassWrapper } = props;
+
+    return (
+        <div className={checkboxesClassWrapper}>
+            {
+                checkboxElements.map(checkbox => {
+                    const { text, ...rest } = checkbox;
+                    return <FormControlLabel
+                        key={text}
+                        control={
+                            <Checkbox
+                                size='small'
+                                name='checked'
+                                color='primary'
+                                {...rest}
+                            />
+                        }
+                        label={text}
+                    />
+                })
+            }
+        </div>
+    );
+};
+
+export default CustomCheckbox;
+
+interface CheckboxElement extends CheckboxProps {
```

Comment:
You are not using the interface in this file

Reply:
It is used under the props